packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -333,7 +333,75 @@ class ShaderMask extends SingleChildRenderObjectWidget {
   }
 }
 
-/// A widget that applies a filter to the existing painted content and then paints [child].
+/// A widget that applies a filter to the existing painted content and then
+/// paints [child].
+///
+/// The filter will only be applied to the area of the background in which the
+/// [child] (or one of its descendants) is actually going to paint in regardless
+/// of the actual size of [child].
+///
+/// In more technical terms, [BackdropFilter] will only apply the filter to the
+/// area of the background that is covered by the [Scene] defined by [child]
```

Comment:
there's only one Scene per screen. This probably means Picture or Layer or some such.

Reply:
I'll consult with the skia team to figure out what exactly defines this area.

Reply:
After consulting with @liyuqian I'll just remove this sentence. The definition above should explain it very well.